blueocean-events/src/main/java/io/jenkins/blueocean/events/BlueMessageEnricher.java

```diff
@@ -96,16 +96,13 @@ public void enrich(@Nonnull Message message) {
             if (message.containsKey("job_run_queueId") && jobChannelItem instanceof hudson.model.Job) {
                 final long queueId = Long.parseLong(message.get("job_run_queueId"));
```

Comment:
`message.get("job_run_queueId")` can return null resulting in `NumberFormatException`. a null check of `message.get("job_run_queueId")` should be added as well since we are fixing this part of the code anyways.